src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceIntellisense/KustoIntellisenseHelper.cs

```diff
@@ -324,5 +341,110 @@ public static async Task<GlobalState> AddOrUpdateDatabaseAsync(IDataSource dataS
             return globals;
         }
 
```

Comment:
These were moved from KustoDataSource to KustoIntellisenseHelper since they're related to that and didn't _need_ to be on KustoDataSource